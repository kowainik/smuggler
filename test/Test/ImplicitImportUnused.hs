module Test.ImplicitImportUnused where

import Data.List

foo = id
