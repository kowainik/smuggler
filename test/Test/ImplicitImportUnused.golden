module Test.ImplicitImportUnused where

foo = id
